react/__tests__/components/ProductImages.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { render } from 'intl-helper'
+import ProductImages from './../../ProductImages'
+
+describe('<ProductImages />', () => {
+  const renderComponent = customProps => {
+    return render(<ProductImages {...customProps} />)
+  }
+
+  it('should be mounted', () => {
+    const { asFragment } = renderComponent()
+    expect(asFragment()).toBeDefined()
```

Comment:
No need for this test

Reply:
In fact, if the component is null or undefined it will fail.

Reply:
You know that `asFragment()` always returns a `<DocumentFragment><DocumentFragment />` right?